apps/preview/next/pages/showcases/MegaMenu/BaseMegaMenu.tsx

```diff
@@ -206,15 +206,17 @@ export default function BaseMegaMenu() {
                         <hr className="mb-3.5" />
                         <ul>
                           {items.map((item) => (
-                            <SfListItem size="sm" role="none">
-                              <a
-                                key={item.title}
-                                className="focus-visible:outline focus-visible:rounded-sm"
+                            <li key={item.title}>
+                              <SfListItem
+                                as="a"
+                                size="sm"
+                                role="none"
                                 href={item.link}
+                                className="focus-visible:outline focus-visible:rounded-sm"
```

Comment:
This PR introduced these classes to the SfListItem component already. It should be applied automatically when you make this component interactive using `as="a"` prop. Same in Vue version.